include/seahorn/Bmc.hh

```diff
@@ -117,10 +171,10 @@ public:
   std::vector<SymStore> &getStates() { return m_states; }
 };
 
+template <class Engine, class Model>
```

Comment:
Move into `BmcImpl.hh` header. Eventually, we will make it a subclass of some general `BmcTrace` class. For now, moving into a header file will suffice.